### PR TITLE
ENDOC-104 nicola feedback

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

File: vuepress/docs/v6.3/docs/getting-started/README.md [hunk 1]
```diff
@@ -168,8 +168,8 @@ sudo kubectl create -f dist/crd
 
 #### Create Namespace
 
-::: tip
-[Kubernetes supports multiple virtual clusters backed by the same physical cluster. These virtual clusters are called namespaces.](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+::: tip What are namespaces?
```
Comment (joewhite101) [inline vuepress/docs/v6.3/docs/getting-started/README.md:171]: capitalize to match others

File: vuepress/docs/v6.3/docs/getting-started/README.md [hunk 2]
```diff
@@ -205,9 +205,8 @@ database connection.
 
 #### Configure Access to Your Cluster
 
-::: tip
-Entando sets up `Ingresses` in Kubernetes to access services from outside your server cluster.
-We'll use this to access Entando from a local browser. 
+::: tip What about networking?
```
Comment (joewhite101) [inline vuepress/docs/v6.3/docs/getting-started/README.md:208]: Capitalize to match others or lowercase others

**Review by joewhite101 — APPROVED** [review]
One small comment on capitalization consistency otherwise good to go.

File: vuepress/docs/v6.3/docs/getting-started/README.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/v6.3/docs/getting-started/README.md]: good catch, will do.

File: vuepress/docs/v6.3/docs/getting-started/README.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/v6.3/docs/getting-started/README.md]: will do

**Pull request merged**